third-party/chromium-synchronization/inspector-issueAdded-types-test.js

```diff
@@ -48,7 +48,6 @@ Array [
   "navigatorUserAgentIssueDetails",
   "quirksModeIssueDetails",
   "sharedArrayBufferIssueDetails",
-  "twaQualityEnforcementDetails",
```

Comment:
Should we add a note to remove this everywhere once the change hits stable?

Reply:
nah let's just kill it now